vms/proposervm/block/block.go

```diff
@@ -102,18 +103,15 @@ func (b *statelessBlock) initialize(bytes []byte) error {
 		return nil
 	}
 
-	cert, err := staking.ParseCertificate(b.StatelessBlock.Certificate)
+	tlsCert, err := x509.ParseCertificate(b.StatelessBlock.Certificate)
 	if err != nil {
 		return fmt.Errorf("%w: %s", errInvalidCertificate, err)
 	}
 
-	if err := staking.VerifyCertificate(cert); err != nil {
```

Comment:
Just reverting diff that we have with avax. Probably was lost in previous merges.